index.html

```diff
@@ -0,0 +1,316 @@
+                    <div class="new-line"></div> The ripples manifested themselves as faint spiral patterns in a bath of microwave radiation that permeates space and preserves a picture of the universe when it was 380,000 years old and as hot as the surface of the sun.
+                    <div class="new-line"></div> Dr. Kovac and his collaborators, working in an experiment known as Bicep, for Background Imaging of Cosmic Extragalactic Polarization, reported their results in a scientific briefing at the Center for Astrophysics here on Monday
+                    and in a set of papers submitted to The Astrophysical Journal.
+                    <div class="new-line"></div>
+                    <div id="inflation-img" class="flex">
+                        <h4>The Theory of Inflation</h4>
+                        <p>Astronomers have found evidence to support the theory of inflation, which explains how the universe expanded so uniformly and so quickly in the instant after the Big Bang 13.8 billion years ago.</p>
+                        <img src="./inflation-945.png" alt="inflation-big-img">
+                        <div class="new-line"></div>
+                        <p>By LARRY BUCHANAN and JONATHAN CORUM</p>
+                    </div>
+                    <div class="new-line"></div> Dr. Kovac said the chance that the results were a fluke was only one in 10 million.
+                    <div class="new-line"></div> Dr. Guth, now 67, pronounced himself “bowled over,” saying he had not expected such a definite confirmation in his lifetime.
+                    <div class="new-line"></div> “With nature, you have to be lucky,” he said. “Apparently we have been lucky.”
+                    <div class="new-line"></div> The results are the closely guarded distillation of three years’ worth of observations and analysis. Eschewing email for fear of a leak, Dr. Kovac personally delivered drafts of his work to a select few, meeting with Dr. Guth,
+                    who is now a professor at Massachusetts Institute of Technology (as is his son, Larry, who was sleeping that night in 1979), in his office last week.
+                    <div class="new-line"></div> “It was a very special moment, and one we took very seriously as scientists,” said Dr. Kovac, who chose his words as carefully as he tended his radio telescopes.
+                    <div class="new-line"></div> Andrei Linde of Stanford, a prolific theorist who first described the most popular variant of inflation, known as chaotic inflation, in 1983, was about to go on vacation in the Caribbean last week when Chao-Lin Kuo, a Stanford
+                    colleague and a member of Dr. Kovac’s team, knocked on his door with a bottle of Champagne to tell him the news.
+                    <div class="new-line"></div>
+                    <iframe width="420" height="315" title="youtube-vid" src="https://youtu.be/ZlfIVEy_YOA">
```

Comment:
### FIX the video not showing
- [ ] The video currently is not showing as expected. Do you mind fixing this?

Below is a comparison of what is expected and what we currently have.

|[Source Page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html)|[Clone Page](https://raw.githack.com/igorkol91/NYTarticle/newbranch/index.html)|
|:--:|:--:|
|![video](https://user-images.githubusercontent.com/22345961/112320988-9668eb80-8cc0-11eb-95bc-c2136e50c5e4.png)|![video-clone](https://user-images.githubusercontent.com/22345961/112321027-9f59bd00-8cc0-11eb-9d4b-32c9602d14b3.png)|